src/main/java/kafdrop/config/ProtobufDescriptorConfiguration.java

```diff
@@ -0,0 +1,58 @@
+package kafdrop.config;
+
+import java.io.File;
+import java.io.FilenameFilter;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.stereotype.Component;
+
+@Configuration
+public class ProtobufDescriptorConfiguration {
```

Comment:
Can we please apply standard formatting, if it's not too much trouble. The rest of the project uses two-space indents.